Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/ActVisualTesting.cs

```diff
@@ -348,7 +353,14 @@ public void CheckSetAppWindowSize()
                     mDriver.ChangeAppWindowSize(0,0);
                     break;
                 case eChangeAppWindowSize.Custom:
-                    mDriver.ChangeAppWindowSize(Convert.ToInt32(GetInputParamCalculatedValue(nameof(SetAppWindowWidth))), Convert.ToInt32(GetInputParamCalculatedValue(nameof(SetAppWindowHeight))));
+                    mDriver.ChangeAppWindowSize(SetAppWindowWidth, SetAppWindowHeight);
```

Comment:
Why 'GetInputParamCalculatedValue' was removed for width and height, are we not providing VE for height and width

Reply:
Added VE support for height and width